paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/utils/FlowControllerTestRunner.kt

```diff
@@ -33,25 +33,46 @@ internal fun runFlowControllerTest(
     resultCallback: PaymentSheetResultCallback,
     block: (FlowControllerTestRunnerContext) -> Unit,
 ) {
-    for (integrationType in FlowControllerTestFactory.IntegrationType.values()) {
```

Comment:
There was something weird here. I'm guessing it had to do with something in the kotlin compiler. But changing it made it run consistently on my machine 🤷